I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 193, in script
    python:
  File "renpy/common/00start.rpy", line 194, in <module>
    renpy.execute_default_statement(True)
  File "game/screens.rpy", line 1891, in execute_default
    default quick_menu = True
Exception: store.quick_menu is being given a default a second time.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/00start.rpy", line 193, in script
    python:
  File "F:\Naiii\APK\RenPY\renpy-8.2.3-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "F:\Naiii\APK\RenPY\renpy-8.2.3-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00start.rpy", line 194, in <module>
    renpy.execute_default_statement(True)
  File "F:\Naiii\APK\RenPY\renpy-8.2.3-sdk\renpy\exports.py", line 4100, in execute_default_statement
    i.execute_default(start)
  File "game/screens.rpy", line 1891, in execute_default
    default quick_menu = True
Exception: store.quick_menu is being given a default a second time.

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.3.24061702
HOLLOW 1.0
Sat Jun 29 22:16:51 2024
